README.md

```diff
@@ -0,0 +1,138 @@
+# New Document# **ALTERNATE BIT PROTOCOL SIMULATOR IMPLEMENTED IN CADMIUM**
+---
+## ORGANIZATION : CARLETON UNIVERSITY
+---
+## AUTHORS :
+### ORIGINALLY DEVELOPED BY : Cristina Ruiz Martin
+### MODIFIED BY : Syed Omar Azam, Mohamed Ali and Ammar Al-Mahbashi                  
+---
+## INTRODUCTION:
+---
+**ABP** (Alternating Bit Protocol) is a communication protocol to ensure reliable transmission through unreliable network. The sender sends a packet and waits for an acknowledgement. If the acknowledgement doesn't arrive within a predefined time, the sender re-sends this packet until it receives an expected acknowledgement and then sends the next packet. In order to distinguish two consecutive packets, the sender adds an additional bit on each packet (called alternating bit because the sender uses 0 and 1 alternatively). This software is a simulator to replicate the behavior of the Alternating Bit Protocol. 

```

Comment:
It would be better if you use Alternation Bit Protocol(ABP) instead of ABP (Alternation Bit Protocol)

Reply:
done. either way, it is ok.